changelog.txt

```diff
@@ -0,0 +1 @@
+feature - Added supported for multi-tenancy for Google Cloud's Identity Platform developers.
```

Comment:
Added multi-tenancy support for Google Cloud's Identity Platform developers.

Reply:
done